install_config/install/deploy_router.adoc

```diff
@@ -498,6 +561,74 @@ $ modshard 3 \
     ROUTE_LABELS='sla=low'
 ----
 
+[[using-namespace-router-shards]]
+=== Using Namespace Router Shards
+
+Each
+link:../../architecture/core_concepts/projects_and_users.html#projects[project has its own namespace].
+The routes for a project can be handled by a selected router by using
+`NAMESPACE_LABELS`.  The router is given a selector for a `NAMESPACE_LABELS`
+label and the project that wants to use the router applies the `NAMESPACE_LABELS`
+label to its namespace.
```

Comment:
I am throughly confused by NAMESPACE_LABELS and ROUTER_LABELS, what routing problems does each address? When do use each and why? How is the customer supposed to think about using these labels? If I am an admin how do I this about this? If I am a user with a project/namespace how do I think about this? How do I work out what will happen to any given path? I thought I understood but reading your comment I don't.

Could you take a shot at wording the above introduction and this example. I can edit the changes.